examples/feature-factory/package.json

```diff
@@ -19,5 +19,11 @@
     "eslint": "^8.53.0",
     "eslint-config-ponder": "workspace:*",
     "typescript": "^5.2.2"
+  },
+  "trustedDependencies": [
```

Comment:
Sorry for backtracking, but I think we actually don't want this. AFAIK Bun is the only package manager that respects `trustedDependencies`. And, once we actually support Bun as a runtime, we won't use `better-sqlite3` at runtime, we will use  `bun:sqlite`.

Reply:
The reason I included this is because this allows for `bunx create-ponder` to function properly. Then running `bun run dev` works because it by default uses node 20.8